src/test/java/org/folio/rest/api/ReindexJobRunnerTest.java

```diff
@@ -78,7 +78,7 @@ public void canReindexInstances() {
 
     // Should be a single reindex message for each instance ID generated in the row stream
     await().atMost(5, SECONDS)
-      .until(FakeKafkaConsumer::getAllPublishedInstanceIdsCount, is(numberOfRecords));
+      .until(FakeKafkaConsumer::getAllPublishedInstanceIdsCount, greaterThanOrEqualTo(numberOfRecords));
```

Comment:
I think it could be useful to explain why this is greater than rather than exact.

As I understand it, it should be exact, however it isn't in practice and we don't know why.